charts/osm/grafana/dashboards/osm-data-plane-performance.json

```diff
@@ -60,7 +60,10 @@
         }
       ]
     },
-    "description": "OSM Data Plane Performance Metrics latest",
+    "title": "OSM Data Plane Performance Metrics",
+    "description": "Compatible with OSM v1.0.0. This dashboard lets you view the performance of OSM’s data plane.",
```

Comment:
nit: Since the description here is used as the revision description, is it possible to specify information like "This dashboard lets you view the performance of OSM’s data plane" somewhere else? Otherwise, this information might make the revision list a bit cluttered.

Reply:
Unfortunately not. In fact, the Grafana dashboard JSON model (documented [here](https://grafana.com/docs/grafana/latest/dashboards/json-model/)) does not even have the `description` field in its JSON. In fact, only the `title` field is documented in the Grafana dashboard JSON model as per Grafana docs (I've searched everywhere and there doesn't seem to be documentation on even the `description` field).

This is how it looks like. The `title` json field corresponds to the title highlighted below, while the `description` field corresponds to the "short description" field highlighted below.

![image](https://user-images.githubusercontent.com/13926417/158427942-f7d9bbf4-dcf5-4c2c-99ae-e9a78f40743a.png)

I think this PR is good to go and could be merged. @jaellio - as I don't have maintainer permissions, can you merge it for me?

Reply:
I cannot merge until there is another review. @snehachhabria would you mind taking another look at this PR?

Reply:
Ok @jaellio this can now be merged.